.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+Contributing Guidelines
+--------------------------------------------------
+
+This document provides general guidelines about how to contribute to the project. Keep in mind these important things before you start contributing.
+
+### Reporting issues
```

Comment:
What about using the headers levels in order? This should be "##", then one deeper "###".